### ISSUE TITLE
new bank accounts and payroll

### ISSUE THREAD

**Issue by uazuaz** [d3athrow/vgstation13#14230]
#### Description of issue
example: you're the HOP and there is a new grown diona that wants a job, so you create a new bank account using the account database and connect it to a new ID for him;
the new account will not receive any money when there is a payroll.
(on a side note, bank account numbers will be 6 digits instead of 5 digits if you create it via account database)

#### Difference between expected and actual behavior
expected all jobs to receive a paycheck;
plant immigrants don't have rights.

#### Steps to reproduce 
create a new bank account, connect it to an ID using the ID computer, wait for a payroll.

#### Length of time in which bug has been known to occur

noticed first time a couple of weeks ago.

#### Server revision

server is down at the moment.

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

**Comment by Kurfursten** [comment]
The only problem is that you could generate money by making a bunch of new accounts and keeping the account info, right?

**Comment by Probe1** [comment]
Yeah that's really fixing an oversight by adding an exploitable oversight.

**Comment by Zth--** [comment]
The problem is that nanotrasen is paying those checks instead of the station account

**Comment by Probe1** [comment]
I don't understand what you mean.

**Comment by Zth--** [comment]
Payrolls are magic numbers. The money is not extracted from the station account. It doesn't matter if the station has 0 money

**Comment by yclatious** [comment]
Id figure that Central itself has its own accounting and not that the entirety of Nanotrasens funding is based on the station or that they would let the station itself handle payments, only giving the station a supply of money for the sake of local trading or emergency. Sounds rational to me.

**Comment by Durgen** [comment]
it would be cool if Traders had bank accounts

**Comment by Intigracy** [comment]
I think having the captain/hop in charge of individual wages and wages coming out of the station account would be neat, with the occasional bump to it.

**Comment by uazuaz** [comment]
can't really understand your concerns, this is one of the many game mechanics that COULD be abused, but:

- to access the account database and create new accounts you need high access (captain and HOP only maybe?)
- the account database is inside the HOP office, so you also need to do grand trespass and risk to be caught when creating the accounts
- to get the money that you collect using multiple account, you need to access all the accounts via ATM using their respective account number and pins, and this is long and boring
- many people don't even know how bank accounts work and about the account database existence
- we don't even have a real economy and money is not so important

TL;DR
nobody is going to bother and waste their time doing this.

additionally, if admins think this is a problem, an IC way to resolve this is that centcomm could express its concerns to the HOP or captain asking why there are so many bank accounts for people that don't appear on the crew manifest and even deleting them.

by the way this matter is not so important, if you want to leave it as is it now it's not a problem at all; i just noticed that it doesn't work as it should and reported it.

**Comment by Kitfox88** [comment]
seems like a fun chance for some adminbus IAA shenanigans if someone does it, yeah.

**Comment by Kurfursten** [comment]
I don't imagine admins taking action to solve this situation IC any more than they take action IC when a HOP gives themselves security access.

**Comment by uazuaz** [comment]
it's fine, it's not gonna happen anyway.
